build/tests/integer-key.json

```diff
@@ -0,0 +1,49 @@
+[
+    {
+        "MethodName": "create",
+        "PositionalParameters": ["a", 0],
+        "expected": "#t"
+    },
+    {
+        "MethodName": "inc",
+        "PositionalParameters": ["a", 1],
+        "expected": "#t"
+    },
+    {
+        "MethodName": "inc",
+        "PositionalParameters": ["a", 5],
+        "expected": "#t"
+    },
+    {
+        "MethodName": "dec",
+        "PositionalParameters": ["a", 2],
+        "expected": "#t"
+    },
+    {
+        "MethodName": "get-value",
+        "PositionalParameters": ["a"],
+        "expected": "4"
+    },
+    {
+        "MethodName": "create",
+        "PositionalParameters": ["b", 5],
+        "expected": "#t"
+    },
+    {
+        "MethodName": "escrow",
+        "PositionalParameters": [
+            "b",
+            "-----BEGIN PUBLIC KEY-----\\nMFYwEAYHKoZIzj0CAQYFK4EEAAoDQgAEz9s2L5K+0ikrWbh0JrzdkYyOPnqZb/Ui\\n/3CKAAxrBVeZ/As6fBCiGjOdtq5Nlwk17+rc/EcDWrskGtb7YvzMIw==\\n-----END PUBLIC KEY-----"
+        ],
+        "expected": "#t"
+    },
+    {
+        "MethodName": "escrow-attestation",
+        "PositionalParameters": ["b"],
+        "expected": "\\(\\(\\(key \"b\"\\) \\(value 5\\) \\(owner \"[^\"]+\"\\)\\) \\(\\(\"[^\"]+\" \"[^\"]+\"\\)\\) \"[^\"]+\"\\)"
```

Comment:
from a symmetry perspective i would have expected results would also be converted to the lingua franca, json. Of course, at some level it's application logic and so it could do what it wants, but in that case we also wouldn't really have to jsonize the input arguments ...?

Reply:
coming in the next big PR